Project.toml

```diff
@@ -5,6 +5,8 @@ version = "0.7.4"
 [deps]
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
```

Comment:
Can you add a corresponding entry in the compat section? That's required for package registration and avoids that arbitrary, possibly incompatible, versions of FillArrays can be installed.

Reply:
Eg. FillArrays = "1" should be fine.